.github/tailor.yaml

```diff
@@ -2,9 +2,9 @@ rules:
   - name: commit title
     description: all commit titles must have a scope, be lowercase, and be no more than 50 characters
     expression:  |-
-      .commits all(((.message test "^[a-z/*-]+: [[:alnum:] -'./-]+") and (.message test "^.{0,50}\\n")) or (.message test "^Revert"))
+      .commits all(((.title test "^[a-z/*-]+: [[:alnum:] -'./-]+$") and (.title length < 51)) or (.title test "^Revert"))
 
-  - name: commit body
-    description: all commits must have a message body with each line having no more than 72 characters
+  - name: commit description
+    description: all commits must have a description with each line having no more than 72 characters
     expression: |-
-      .commits all((.message test "^.*\\n\\n.{10,72}(\\n.{0,72})*$") or (.message test "^Revert"))
+      .commits all((.description lines all(. length < 73) and (.description lines length > 0)) or (.title test "^Revert"))
```

Comment:
Do blank lines not get added into the description string array? Otherwise would you be able to avoid the `.description lines length > 0` check via something with just multiple blank lines like:
```
example: my commit message



```

Reply:
Yes, you could get around it by doing that. You would have to go out of your way to trick it, so I'm fine depending on developers to catch this.